server/src/main/java/com/objectcomputing/checkins/services/file/GoogleDriveAccessor.java

```diff
@@ -16,7 +16,7 @@
 @Singleton
 public class GoogleDriveAccessor {
 
-    private static final JsonFactory JSON_FACTORY = JacksonFactory.getDefaultInstance();
+    private static final JsonFactory JSON_FACTORY = GsonFactory.getDefaultInstance();
```

Comment:
Any particular driver for this change? Just curious.

Reply:
They removed JacksonFactory in favor of GsonFactory 

https://cloud.google.com/java/docs/reference/google-http-client/latest/com.google.api.client.json.jackson2.JacksonFactory